docker-compose/docker-compose.yml

```diff
@@ -14,7 +12,7 @@ version: "2"
 
 services:
     alfresco:
-        image: alfresco/alfresco-content-repository:6.2.0-A3
+        image: quay.io/alfresco/alfresco-content-repository:latest
```

Comment:
Sorry, we don't allow floating tags. Have you tried 6.2.0-A4? If it doesn't contain the changes you need, we can schedule another release.

Reply:
I understand, we need a 6.2.0-A5 release then, which has the latest changes, including the Sync service AMP.

Reply:
If it's ok with I can trigger release for 6.2.0-A5.

Reply:
Do you think it's possible to release 6.2.0-A5 today so I can merge the PR before I leave on vacation? :smiley: